linetools/spectra/io.py

```diff
@@ -236,12 +236,24 @@ def readspec(specfil, inflg=None, efil=None, verbose=False, flux_tags=None,
                 tmpsig[gdp] = np.sqrt(1./sig[gdp])
                 sig = tmpsig
                 wave = 10.**wave
-    elif (head0['NAXIS'] == 2) and (head0['NAXIS2'] == 5): # SDSS .fit format
-        fx = hdulist[0].data[0,:].flatten()
-        sig = hdulist[0].data[2,:].flatten()
-        wave = setwave(head0)
+    elif head0['NAXIS'] == 2:
```

Comment:
Is there a FITS header keyword that can be used to show this is a DESI brick, instead of the file structure? Using a keyword seems less likely to produce unexpected bugs with unknown file types.